src/components/display/components/DropdownComponent.tsx

```diff
@@ -0,0 +1,117 @@
+import { MaterialCommunityIcons } from '@expo/vector-icons'
+import { DropdownArgs, DropdownOption } from 'display-api'
+import * as React from 'react'
+import { View, StyleSheet } from 'react-native'
+import { Menu, Provider, Text, TouchableRipple, ActivityIndicator } from 'react-native-paper'
+import colors from '../../../utils/colors'
+import { globalStyles } from '../../../utils/styles'
+import { getComponentIcon } from '../../../utils/utils'
+import { blockMethod, setMethodVariable } from '../method'
+
+export const DropdownComponent = ({
+	color_scheme,
+	disabled,
+	readonly,
+	name,
+	on_change,
+	options,
+	variant,
+}: DropdownArgs) => {
+	const [isVisible, setVisible] = React.useState(false)
+	const [value, setValue] = React.useState(name)
```

Comment:
This should probably be the `default` arg, or 0.

e.g if the options are Open, Doing, Done, then having default `0` should default the dropdown to Open.

Reply:
Is `default` a valid argument name? I couldn't use it because it's a reserved word. Can we change it to something else?

Reply:
Ah yeah it is. You can use `default: initial` when destructuring the DropdownArgs to rename it, that should work